app/pages/SiloAccessPage.tsx

```diff
@@ -162,6 +164,12 @@ export function SiloAccessPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Access24Icon />}>Access</PageTitle>
+        <DocsPopover
+          heading="Access"
+          icon={<Access16Icon />}
+          summary="Silo-level access roles and permissions. Permissions set at the silo level cascade down to all projects within the silo."
+          links={[docLinks.keyConceptsIam, docLinks.access]}
+        />
```

Comment:
This and project access should be more similar. Since "permissions" is not really an API concept, I would avoid it.

> Set silo-level roles for users and groups. Silo roles apply to all projects within the silo.

I wonder if we should also mention that you can override these with project roles. That might be too much, and covered by the docs. The problem is you also have to explain that you can only override if the role is stronger.